.github/workflows/conformance-aws-cni.yaml

```
@@ -138,10 +138,34 @@ jobs:
 
       - name: Create EKS cluster without nodegroup
         run: |
-          eksctl create cluster \
-            --name ${{ env.clusterName }} \
-            --tags "usage=${{ github.repository_owner }}-${{ github.event.repository.name }},owner=${{ steps.vars.outputs.owner }}" \
-            --without-nodegroup
+          cat <<EOF > eks-config.yaml
+            apiVersion: eksctl.io/v1alpha5
+            kind: ClusterConfig
+
+            metadata:
+              name: ${{ env.clusterName }}
+              region: us-west-2
```

It is it a mandatory metadata, or does it default to the default configured version if not provided? If it does, I suggest we simply remove it. If it does not:
```suggestion
              region: ${{ env.region }}
```